src/controllers/StudentController.js

```diff
@@ -69,7 +69,11 @@ module.exports = {
     try {
       const student = await Student.findByPk(req.params.id);
       if (student) {
-        const updatedStudent = await student.update(req.body);
+        let changedStudent = req.body;
+        if (changedStudent.status === 'Отчисленный') {
```

Comment:
```suggestion
        if (changedStudent.status === statuses.expelled) 
```
Предлагаю статусы хранить в отдельном файле и обращаться к ним.

Reply:
Тогда надо новый тикет создавать на изменение в модели Студенты поля enum, где вкладывать список по статусам, делать новый файл миграции и т.д. В этом тикете цель только проверка статуса Отчисленный и передача groupId = null.